src/Microsoft.AspNetCore.Mvc.Core/ApiExplorer/ApiConventionTypeMatchBehavior.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.ApiExplorer
+{
+    /// <summary>
+    /// The behavior for matching the name of a convention parameter.
+    /// </summary>
+    public enum ApiConventionTypeMatchBehavior
```

Comment:
Might be valuable to throw an "Exact" in here for those who want to be extra explicit. Also lets us indicate what the default behavior is.

Reply:
> #7939 (comment) for Exact

Ah, I'm with you. I think `Exact` is valuable :/